docs/VnVPlan/VnVPlan.tex

```diff
@@ -900,18 +1045,18 @@ \subsection{Traceability Between Test Cases and Requirements}
   UH-UP1 & \\ \hline
   UH-UP2 & \\ \hline
   UH-AS1 & \\ \hline
-  PR-SL1 & \\ \hline
+  PR-SL1 & TC-PR-1\\ \hline
   PR-SL2 & \\ \hline
```

Comment:
this requirement can be removed from the SRS, so a strikeout of it in the SRS like Rafeed did earlier https://github.com/Spitgranger/SyncMaster/pull/221